x-pack/plugins/index_lifecycle_management/public/application/services/policies/cold_phase.ts

```diff
@@ -53,9 +53,9 @@ export const coldPhaseFromES = (phaseSerialized?: SerializedColdPhase): ColdPhas
         Object.entries(allocate.require).forEach((entry) => {
           phase.selectedNodeAttrs = entry.join(':');
         });
-        if (allocate.number_of_replicas) {
-          phase.selectedReplicaCount = allocate.number_of_replicas.toString();
-        }
+      }
```

Comment:
Hi @jloleysens , I don't think `number_of_replicas` needs to depend on `require` property of `AllocateAction`, just wanted to confirm with you.